.github/CODEOWNERS

```diff
@@ -43,6 +43,7 @@ Lib/test/test_patma.py        @brandtbucher
 Lib/test/test_type_*.py       @JelleZijlstra
 Lib/test/test_capi/test_misc.py  @markshannon @gvanrossum
 Tools/c-analyzer/             @ericsnowcurrently
+Lib/_pyrepl/*                 @pablogsal @lysnikolaou @ambv
```

Comment:
Let's stick it before `Lib/test/...` so it's kind of sorted, can help avoid conflicts.

Reply:
Done